apps/web/src/pages/api/og/dao.tsx

```diff
@@ -0,0 +1,136 @@
+import { ImageResponse } from '@vercel/og'
+import { getFetchableUrl } from 'ipfs-service/src/gateway'
+import { NextRequest } from 'next/server'
+
+import { DaoOgMetadata } from 'src/data/contract/requests/getDaoOgMetadata'
+import NogglesLogo from 'src/layouts/assets/builder-framed.svg'
+
+export const config = {
+  runtime: 'edge',
+}
+
+const ptRootRegular = fetch(
+  new URL('public/fonts/pt-root-ui_regular.ttf', import.meta.url)
+).then((res) => res.arrayBuffer())
+
+const ptRootMedium = fetch(
+  new URL('public/fonts/pt-root-ui_medium.ttf', import.meta.url)
+).then((res) => res.arrayBuffer())
+
+const ptRootBold = fetch(
+  new URL('public/fonts/pt-root-ui_bold.ttf', import.meta.url)
+).then((res) => res.arrayBuffer())
+
+export default async function handler(req: NextRequest) {
+  const { searchParams } = new URL(req.url)
+  const rawData = searchParams.get('data')
+
+  if (!rawData) return new Response(undefined, { status: 400 })
+
+  const data: DaoOgMetadata = JSON.parse(rawData)
+
+  const [ptRootRegularData, ptRootMediumData, ptRootBoldData] = await Promise.all([
+    ptRootRegular,
+    ptRootMedium,
+    ptRootBold,
+  ])
+
+  const daoDataWithLabel = (label: string, data: string) => {
+    return (
+      <div
```

Comment:
nit: we can use zord components `<Stack />` here -- not sure if you're committed to keeping zord as the UI library, but it is useful in reducing the amount of css styles that get generated on build. Using zord components and their props reuses the same class names keeping allow the amount of styles to plateau over time

Reply:
vercel/og only works a subset of css styles which would cause incompatibilities with zord I think. It also requires using the edge runtime which was causing issues with some zord dependencies when I was developing things. an alternate solution would be to build pages with zord components and use puppeteer to screenshot but last time I tried that it was causing capacity issues with nextjs lambdas. can try to make this work too if you think it would be better to have consistency with these though! let me know what you think